src/store/modules/users.js

```diff
@@ -73,7 +73,7 @@ const mutations = {
   SET_CURRENT_ORG(state, org) {
     // 系统组织不设置成 Pre org
     const currentOrg = state.currentOrg
-    if (!currentOrg?.autoEnter && !currentOrg.is_system) {
+    if (currentOrg && !currentOrg.autoEnter && !currentOrg.is_system) {
       state.preOrg = currentOrg
       setPreOrgLocal(state.username, currentOrg)
     }
```

Comment:
The given code does not contain any known defects, inconsistencies, or optimizations that might suggest an issue. However, for consistency and clarity improvement:

* The variable 'state' has already been declared with a lowercase letter at the beginning to align properly as a global object in JavaScript.

Please let me know if there is anything more specific you need help with!